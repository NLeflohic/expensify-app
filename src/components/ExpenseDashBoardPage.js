import React from "react";
import ExpenseTotal from "./ExpenseTotal";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";

const ExpenseDashBoardPage = () => (
  <div>
    <ExpenseTotal />
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default ExpenseDashBoardPage;
