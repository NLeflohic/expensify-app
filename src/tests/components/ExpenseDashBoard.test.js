import React from "react";
import { shallow } from "enzyme";
import ExpenseDashBoardPage from "../../components/ExpenseDashBoardPage";

test("Should generate ExpenseDashBoard", () => {
  const wrapper = shallow(<ExpenseDashBoardPage />);
  expect(wrapper).toMatchSnapshot();
});
