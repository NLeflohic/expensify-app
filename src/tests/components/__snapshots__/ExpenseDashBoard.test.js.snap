// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should generate ExpenseDashBoard 1`] = `
<div>
  <Connect(ExpenseTotal) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
